src/pybind/mgr/rgw/module.py

```diff
@@ -181,6 +191,27 @@ def _cmd_rgw_realm_reconcile(self,
 
         return HandleCommandResult(retval=retval, stdout=out, stderr=err)
 
+    def serve(self) -> None:
+        self.run = True
+
+        self.log.info('Starting usage trim loop')
+        while self.run:
+            if self.usage_trim_older_than_days <= 0:
+                self.log.info('Skipping usage trim because usage_trim_older_than_days is not > 0')
+                self.event.sleep(self.usage_trim_interval)
```

Comment:
What happens if somebody configure this parameter to a very big value? the thread could be sleeping here for a very big amount of time and if the user puts a smaller value the new configuration parameter won't be applied!

Reply:
Good feedback! What do we think about a good backoff sleep for when it's not configured? is one minute to small?

Reply:
In terms of sleep I think 1 min should be OK, probably we have to document that so the user is aware that new values of configuration may take some time to take effect.